shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rule/EncryptTable.java

```diff
@@ -196,4 +206,5 @@ public Optional<Boolean> getQueryWithCipherColumn() {
     public Optional<EncryptColumn> findEncryptColumn(final String logicColumn) {
         return Optional.ofNullable(columns.get(logicColumn));
     }
+    
```

Comment:
Is it a new added blank line? How about to remove it?

Reply:
ok